lib/internal/process/warning.js

```diff
@@ -129,7 +125,7 @@ function emitWarning(warning, type, code, ctor) {
     if (process.throwDeprecation)
       throw warning;
   }
-  process.nextTick(doEmitWarning(warning));
+  process.nextTick(() => process.emit('warning', warning));
```

Comment:
How about reusing the function instead of creating a new anonymous function every time? For example:

```js
function doEmitWarning(warning) {
  process.emit('warning', warning);
}

// ...

process.nextTick(doEmitWarning, warning);
```

Reply:
I get what your meaning, closing

Reply:
nice idea, thanks